package com.java.exception;

public class ExceptionTest3 {

	public static void main(String[] args) {
		String s = null;
		System.out.println(s.length());
		System.out.println("这里会被执行吗？");
	}

}
